packages/sql/src/SQLManager.ts

```diff
@@ -0,0 +1,10 @@
+import RequestManager from './RequestManager';
```

Comment:
Is this class still being used? It looks like it is not anymore

Reply:
Nope, it's an early test